addons/mass_mailing/models/mailing.py

```diff
@@ -952,14 +962,14 @@ def _action_send_statistics(self):
                     'mailing_type': mailing_type,
                 },
             )
-
             rendered_body = self.env['ir.qweb']._render(
                 'digest.digest_mail_main',
                 {
                     'body': tools.html_sanitize(link_trackers_body),
                     'company': mail_company,
                     'user': mail_user,
                     'display_mobile_banner': True,
+                    'mailing_off_token': self._get_unsubscribe_token(),
```

Comment:
what if a portal user is set as the responsible of the mailing ? (it makes not a lot of sense but) he will be able to disable the feature... I would give the token only to the admin (or at least to the mailing user)

Reply:
Perhaps this can be discussed with lna when he's Testing... not much was specified in the task description

Reply:
changed it so only users in `group_mass_mailing_user` receive the link